LICENSE

```diff
@@ -230,6 +230,9 @@ http://prestodb.io
 developed by Facebook (https://github.com/facebook/presto)
 License: Apache License v2 (http://www.apache.org/licenses)
 
+FIXME: I cannot find this source code in our Git repository. am I correct in
+thinking this is no longer used or is a build time inclusion?
```

Comment:
@mxm can you weigh in?

Reply:
AFAIK the only left-over is the Presto parser grammar which our parser is based off, i.e. https://github.com/prestodb/presto/blob/master/presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4